embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/prefs/PreferencesService.kt

```diff
@@ -52,17 +52,6 @@ interface PreferencesService {
      */
     var permanentSessionProperties: Map<String, String>?
 
-    /**
-     * No longer used, will be removed in a future version.
-     *
-     * Method is still present to ensure that during any upgrades to SDK3, any custom
-     * personas are merged with the user personas list.
-     *
-     * @return custom personas
-     */
-    @Deprecated("")
```

Comment:
Wait, this means we can remove the entire personas feature?

Reply:
Not quite. Custom personas is something that used to exist in a very old version of the SDK & was no longer settable via our API, but we kept reading it in case folks had persisted the value. I think it's safe to remove this now as enough time has passed & few people use personas anyway

Reply:
OK, that's different than what is done when customers call `addPersona()`? In that case, I spoke too soon..... but now that I've said it, I still want to deprecate lol